src/modules/pokeRender.js

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable no-await-in-loop */
```

Comment:
- [ ] (OPTIONAL) Next time, it would be great if you do not disable any linters checks but instead search for a better solution online 💯